doc/source/cluster/autoscaling.rst

```diff
@@ -0,0 +1,111 @@
+.. _ref-autoscaling:
+
+Cluster Autoscaling
+===================
+
+Basics
+------
+
+The Ray Cluster Launcher will automatically enable a load-based autoscaler. When cluster resource usage exceeds a configurable threshold (80% by default), new nodes will be launched up the specified ``max_workers`` limit (in the cluster config). When nodes are idle for more than a timeout, they will be removed, down to the ``min_workers`` limit. The head node is never removed.
+
+The default idle timeout is 5 minutes, which can be set in the cluster config. This is to prevent excessive node churn which could impact performance and increase costs (in AWS / GCP there is a minimum billing charge of 1 minute per instance, after which usage is billed by the second).
+
+The basic autoscaling config settings are as follows:
+
+.. code::
+
+    # An unique identifier for the head node and workers of this cluster.
+    cluster_name: default
+
+    # The minimum number of workers nodes to launch in addition to the head
+    # node. This number should be >= 0.
+    min_workers: 0
+
+    # The autoscaler will scale up the cluster to this target fraction of resource
```

Comment:
```suggestion
    # The autoscaler will scale up the cluster to this target fraction of CPU
```
Not sure if this is accurate.

Reply:
It's actually resource usage, not CPU

Reply:
Could we specify what resources are considered?

Reply:
I've updated this in the paragraph above.